OpenSim/Simulation/Model/Model.h

```diff
@@ -522,7 +522,21 @@ OpenSim_OBJECT_NONTEMPLATE_DEFS(Model, ModelComponent);
     GeneralForceSubsystem allocated by this %Model. **/
     SimTK::GeneralForceSubsystem& updForceSubsystem() 
     {   return *_forceSubsystem; }
-
+    /** (Advanced) Get read only access to internal Simbody RigidBodyForces at Dynamics stage **/
+    const SimTK::Vector_<SimTK::SpatialVec>& getRigidBodyForces(const SimTK::State& state)
```

Comment:
Does `SimTK::Vector_<SimTK::SpatialVec>` get wrapped properly in the bindings? I don't see a `%template` for it.

Reply:
Yes it does here's the signature from the newly generated Model.java
public VectorOfSpatialVec getGravityBodyForces(State state) {
  ....
}

Reply:
Should we create our own SWIG template to follow the convention for other vectors like `VectorVec3`? Perhaps `VectorSpatialVec`? I definitely wouldn't know to search for the signature `VectorOfSpatialVec` without knowing it existed already. (I'm assuming `VectorOfSpatialVec` is auto-generated by SWIG).